actors/evm/tests/call.rs

```diff
@@ -403,14 +403,18 @@ push4 0xffffffff
 # call_actor must be from delegatecall
 delegatecall
 
-# copy result to mem 0x00 (overwrites input data)
+# write exit code memory
+push1 0x00 # offset
+mstore8
+
 returndatasize
-push1 0x00
-push1 0x00
+push1 0x00 # offset
+push1 0x01 # dest offset
```

Comment:
???

Reply:
first byte is the exit code pushed put into memory

Reply:
^what melanie said -- i can add comments making clear.